### ISSUE TITLE
Some stripe in predict depth map when using original model and dataset

### ISSUE THREAD

**Issue by stevenhxy** [ashesknight/tof-mpi-remove#2]
Hello gtdong

Sorry to bother you.
May I borrow your 3 mins for one question? Thanks.

I used your github code, TB dataset and AI model(sample_pyramid_add_kpn) on google drive. (no modification)
And I run "python start.py -m sample_pyramid_add_kpn -p size384 -k depth_kinect_msk -t TB -i 239 320 -o mean_l1 -g 1 -f output -c 200000" 

But I get a little wired result (predict depth map) as following. 
There are some white stripe in the red circle, while there is no white stripe in your paper figures.
![2_notes](https://user-images.githubusercontent.com/84314818/208831827-32ad4bfb-bc0d-4e52-9aab-f7de020eba79.png)
![image](https://user-images.githubusercontent.com/84314818/208832784-3efa782f-e8f1-4f84-aa36-359d29b5d7d0.png)

Do you know the reason and how to fix this problem?
Looking forward to your kind reply.
Best regard,

Thanks, 
Steven

**Comment by YanXiaoLin** [comment]
Hello gtdong
Sorry to bother you.
May I borrow your 3 mins for one question? Thanks.
I run “python start.py -b 1 -s 200000 -m sample_pyramid_add_kpn -p size224 -k depth_kinect_msk -l 0.0004 -t TB -i 239 320 -o mean_l1 --addGradient sobel_gradient -g 1 -e 1200”.
But the following error prompts that it cannot run.
INFO:tensorflow:loss = 0.3420182, step = 0
ERROR:tensorflow:Model diverged with loss = NaN.
Do you know the reason and how to fix this problem?
Looking forward to your kind reply.
Best regard,
Thanks,
Yan
